lib/Migration/Version020400Date20250203125359.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+declare(strict_types=1);
+/**
+ * SPDX-FileCopyrightText: 2025 Nextcloud GmbH and Nextcloud contributors
+ * SPDX-License-Identifier: AGPL-3.0-or-later
+ */
+
+namespace OCA\Assistant\Migration;
+
+use Closure;
+use OCP\DB\ISchemaWrapper;
+use OCP\DB\Types;
+use OCP\Migration\IOutput;
+use OCP\Migration\SimpleMigrationStep;
+
+class Version020400Date20250203125359 extends SimpleMigrationStep {
+
+	/**
+	 * @param IOutput $output
+	 * @param Closure(): ISchemaWrapper $schemaClosure
+	 * @param array $options
+	 * @return null|ISchemaWrapper
+	 */
+	public function changeSchema(IOutput $output, Closure $schemaClosure, array $options): ?ISchemaWrapper {
+		/** @var ISchemaWrapper $schema */
+		$schema = $schemaClosure();
+		$schemaChanged = false;
+
+		if ($schema->hasTable('assistant_chat_msgs')) {
+			$table = $schema->getTable('assistant_chat_msgs');
+			if (!$table->hasColumn('task_id')) {
+				$table->addColumn('task_id', Types::BIGINT, [
+					'notnull' => true,
+					'default' => 0,
+					'unsigned' => true,
+				]);
+				$schemaChanged = true;
+			}
```

Comment:
wdyt about making it unique and adding an index here for just the task_id?

Reply:
:+1: I'll add an index. I don't know now which one, depending on the decision we make on your other comment.

Reply:
index added